Shuckle Imagemaker fix

Shidouuu opened the pull request:
Everything should work perfectly now.

Also, a minor update to requirements.txt for modules that I've used and have worked, minus simplejson since the newer version was just a bug fix.

Additionally, I changed some if's to elif's in Shuckbot.py.

Shidouuu commented on modules/Shuckbot.py:
```diff
@@ -199,19 +199,21 @@ async def on_message(message):
 
-    if message.clean_content.lower() == "b" or message.clean_content.lower() == "n":
+        elif content.lower().startswith(("shuck")):
+            await imagefun.shuckle_imagemaker(message)
```
Oops, forgot to press enter. Still works though

Shidouuu commented on modules/Shuckbot.py:
```diff
@@ -199,19 +199,21 @@ async def on_message(message):
-    if message.clean_content.lower() == "b" or message.clean_content.lower() == "n":
+        elif content.lower().startswith(("shuck")):
+            await imagefun.shuckle_imagemaker(message)
+    elif message.clean_content.lower() == "b" or message.clean_content.lower() == "n":
```
Oops, forgot to press enter. Still works though

billofbong commented:
Make these edits in the base Shuckbot.py file not modules/Shuckbot.py—that file doesn’t exist in production and was erroneously placed in the repo

Shidouuu commented:
Whoops, didn't notice that. Changes have been made.

Shidouuu commented:
I added Hirtsifier functionality: `;hirtsifier <text>`

Also, I changed the based elif statement back to an if and moved it to the bottom so it will still activate when using another command, ex. `@ShuckBot based`
